### PR TITLE
[android] Added Google advertising id docs

### PR DESCRIPTION
**Pull request by radixdev**
# Pull Request/Issue Resolution

**Description of Change:**
Docs for Google Advertising ID

**Reason for Change:**

Closes https://jira.braze.com/browse/PC-1043

---
---

## PR Checklist
- [x] Ensure you have completed our CLA.
- [ ] Tag @EmilyNecciai as a reviewer when the your work is done and ready to be reviewed for merge. 
- [x] Consult the [Docs Text Formatting Guide](https://github.com/Appboy/success/wiki/Docs-Text-Formatting-Guide) to be sure you're utilizing the proper markdown formatting.
- [x] Consult the [Docs Writing Style Guide & Best Practices](https://github.com/Appboy/success/wiki/Writing-Style-Guide-&-Best-Practices) to be sure you're aligning with our voice and other style best practices.
- [ ] [Preview your deployed changes](https://homeslice.braze.com/docs) to confirm that none of your changes break production. Pay close attention to links and images.
- [x] Tag others as Reviewers as necessary.
- [ ] If you have modified any links, be sure to add redirects to `config/nginx.conf.erb`.

---
---

### CODE REVIEW

File: _docs/_developer_guide/platform_integration_guides/android/analytics/google_advertising_id.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+search_rank: 5
+---
+
+# Google Advertising ID
+
+The Google Advertising ID is a user-specific, unique, resettable ID for advertising, provided by Google Play services. It gives users better controls and provides developers with a simple, standard system to continue to monetize your apps. It is an anonymous identifier for advertising purposes and enables users to reset their identifier or opt out of interest-based ads within Google Play apps. See [here][2] for more information.
+
+# Passing the Google Advertising ID to Braze
+
+The Google Advertising ID is not automatically collected by the Braze SDK via the [`Appboy.setGoogleAdvertisingId()`][1] method. Note that Google requires the Advertising ID to be collected on a non-UI thread.
```
Comment (Bucimis) [inline _docs/_developer_guide/platform_integration_guides/android/analytics/google_advertising_id.md]: via -> and must be set manually via

Comment (radixdev) [reply _docs/_developer_guide/platform_integration_guides/android/analytics/google_advertising_id.md]: done

**Comment by EmilyNecciai** [comment]
@radixdev Should this go out with tomorrow's release?

**Comment by radixdev** [comment]
@EmilyNecciai yes

**Comment by EmilyNecciai** [comment]
@cla-check

**Comment by EmilyNecciai** [comment]
@cla-bot check

**Pull request merged**